### ISSUE TITLE
Review the process for creating the WSG checklist to design one that is accessible

### ISSUE THREAD

**Issue by jenstrickland** [w3c/sustainableweb-wsg#217]
It would be great if we could schedule to review and discuss the process for creation of an accessible WSG checklist.

Currently, @AlexDawsonUK uses Pages and generates a PDF, but lacks the ability to check and remediate the PDF.

Additionally, the checklist was using a table for layout and lacked the semantics needed for a standards-compliant, accessible PDF.

Regarding a format anyone can use, people can use Google, Pages, Word, etc. At this time, Word is the best tool for creating the original that will deliver an accessible PDF. 

To check and remediate PDF accessibility it’s best to have Adobe Acrobat Pro. There are free tools to be found with a web search, but I’m unsure how helpful they are for remediation.  Remediation in Acrobat Pro is a prehistoric, hair-pulling experience often, but there is nothing to replace it that doesn't require a third-party vendor and budget.

A smoother process might be to create an HTML page with great print styling, which would allow anyone comfortable with HTML to update. It should not use a table for layout, rather it should use semantic HTML.

**Comment by jenstrickland** [comment]
I appreciate your quick turnaround on an approach. There are accessibility issues with both providing a Google Sheet and the structure of the file.

Many in the web accessibility space are less aware of document accessibility practices, and it isn't a quick learning curve.

Hopefully this topic will be a discussion point is an upcoming Sustainable Web Interest Group meeting, so we can discuss in detail.

**Comment by jenstrickland** [comment]
> Correct me if I'm wrong, but if we have accessible HTML, then the corresponding PDF does not need to be "fully" accessible. Same content, different format. (Also, we could remove the PDF because we don't need it.)

That's exactly what I mean. 

The purpose of a checklist is so people could use it, so we'd need to make it print well. People that want a PDF could make one from that, too. 

The HTML page that Alex shared last night is not accessible.

**Comment by AlexDawsonUK** [comment]
I will close this as it has remained dormant for over 6 months and the changes made appear to have been positive.

If further issues occur they can be recorded by opening a new issue.
